sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_shared_access_signature.py

```diff
@@ -65,6 +65,8 @@ def generate_account_sas(
         restricts the request to those IP addresses.
     :keyword str protocol:
         Specifies the protocol permitted for a request made. The default value is https.
+    :keyword str encryption_scope:
```

Comment:
Is encryption scope only support for account SAS? Or is it also supported for the other SASs (file system, directory, and file). If it is supported for the others, we'll need to add that too.

Reply:
Good catch, will add support for the other SAS types.